feature/amazon/build.gradle

```diff
@@ -1,14 +1,24 @@
 plugins {
     alias libs.plugins.android.library
     alias libs.plugins.kotlin.android
-    alias libs.plugins.mavenPublish
+}
+
+if (!project.getProperties()["ANDROID_VARIANT_TO_PUBLISH"].contains("customEntitlementComputation")) {
+    apply plugin: "com.vanniktech.maven.publish"
```

Comment:
I didn't like how many hardcoded strings I had to add here (property name, property value, plugin name... But I think it should be ok? Open to thoughts if anyone has other ideas.

Reply:
can you do the if inside the plugins block? I assume you can't and that's why you put it here.

Also, is it possible to do this?

```
if (!project.getProperties()["ANDROID_VARIANT_TO_PUBLISH"].contains("customEntitlementComputation")) {
	plugins {
	  alias libs.plugins.mavenPublish
	}
}
```

Alternatively, you can do this I think https://stackoverflow.com/a/53408667 , but I don't think it's better

Reply:
I tried putting the if inside the plugins block and it didn't work. Also tried using the `alias` format without luck, so was forced to use the old format :(

Reply:
As for using flavors... That would only help for the condition, but not the format right? As in, we would still have to use `apply plugin....`